e2e/benchmarks/local-benchmark/loader.js

```diff
@@ -29,10 +42,20 @@ function loadScript(url) {
   })
 }
 
+/**
+ * processUrls is called from loadTFJS(localBuild)
+ * processUrls is used to replace the urls in the urls array with local urls 
+ * if the localBuild array includes the package name
+ * 
+ * @param {string array} localBuild
+ * @param {string array} urls
```

Comment:
JSDoc has a syntax for array types: https://jsdoc.app/tags-type.html

Reply:
Thanks. Fixed JSDoc syntax for array types.